internal/backend/user.go

```diff
@@ -29,7 +29,7 @@ type user struct {
 
 	db *db.DB
 
-	states      map[int]*state.State
+	states      map[state.StateID]*state.State
 	statesLock  sync.RWMutex
 	nextStateID int
```

Comment:
why not make this of type `state.StateID`?

Reply:
Good idea. Done.